src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -135,6 +139,200 @@ void RaiseSecond()
             VisualStudio.Editor.Verify.TextContains("Second?.");
         }
 
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.EditorConfig)]
+        [Trait(Traits.Feature, Traits.Features.CodeActionsFixAllOccurrences)]
+        [WorkItem(15003, "https://github.com/dotnet/roslyn/issues/15003")]
+        [WorkItem(19089, "https://github.com/dotnet/roslyn/issues/19089")]
+        public void ApplyEditorConfigAndFixAllOccurrences()
+        {
+            var markup = @"
+class C
+{
+    public int X1
+    {
+        get
+        {
+            $$return 3;
+        }
+    }
+
+    public int Y1 => 5;
+
+    public int X2
+    {
+        get
+        {
+            return 3;
+        }
+    }
+
+    public int Y2 => 5;
+}";
+            var expectedText = @"
+class C
+{
+    public int X1 => 3;
+
+    public int Y1 => 5;
+
+    public int X2 => 3;
+
+    public int Y2 => 5;
+}";
+
+            // CodingConventions only sends notifications if a file is open for all directories in the project
+            VisualStudio.SolutionExplorer.OpenFile(new ProjectUtils.Project(ProjectName), @"Properties\AssemblyInfo.cs");
```

Comment:
:memo: I pinged @AmadeusW regarding this bug. If it doesn't get fixed, the entire .editorconfig experience we finally complete with this pull request will come crashing down.

Reply:
➡️ This bug has been fixed for 15.8.